ydb/library/yql/parser/pg_wrapper/test/ya.make

```diff
@@ -15,7 +15,6 @@ ELSE()
 ENDIF()
 
 REQUIREMENTS(
-    cpu:4
```

Comment:
please don't

Reply:
reverted (it was stated that pg_wrapper tests do work in 16 threads)